backend/src/main/java/planit/people/preparation/Services/Service_User.java

```diff
@@ -41,7 +41,7 @@ public UserCreationResponse create_new_google_account(Entity_GoogleAccount googl
             Entity_GoogleAccount entity_google_account = idaoGoogleAccount
                     .save(new Entity_GoogleAccount(google_account.getEmail(), google_account.getThe_user(), google_account.getRefresh_token()));
             System.out.println("newly created user: " + entity_google_account);
-            return new UserCreationResponse(String.valueOf(entity_google_account.getThe_user().getUser_id()), String.valueOf(entity_google_account.getId()));
+            return new UserCreationResponse(entity_google_account.getThe_user().getUser_id(), entity_google_account.getId());
```

Comment:
Can we also rename `getThe_user` to `getTheUser`?

Reply:
Auto generated getter because the field name is the_user